test/utils/mobx.config.ts

```diff
@@ -0,0 +1,3 @@
+import { useStrict } from 'mobx';
+
+useStrict(false);
```

Comment:
in order not to break everyone's code, after discussion, this should be done in 'horizontal' manner